ui/Makefile

```diff
@@ -9,15 +9,18 @@ node_modules:
 	yarn install
 
 e2e: node_modules
+	yarn generate
 	yarn test:junit
 
 build: node_modules $(UISOURCES)
 	yarn build
 
 lint: node_modules $(UISOURCES)
+	yarn generate
```

Comment:
probably better off making another make target and making that a dep on line 18 here and elsewhere